### ISSUE TITLE
Add a "Packaged by" field

### ISSUE THREAD

**Issue by kekkoudesu** [flathub-infra/website#3125]
I think adding a "Packaged by" field for every app would improve the UI for Flathub.

Here's why:

1. **The user can clearly see the software is developed "by Google", but refi64 is responsible for packaging it.** The "Unverified" tag already communicates that to those who know what it means (the [About page](https://flathub.org/about) explains it), but adding a "Packaged by" field would make it crystal clear.
2. **Packagers should get some recognition!** They make a big difference in user's lives by bringing up-to-date software to them they otherwise wouldn't be able to use, and they make a big difference for upstream developers who don't need to figure out how to write a Flatpak manifest from scratch when they decide to take ownership of the package.

Here's what I think it might look like:

![image](https://github.com/flathub/flathub/assets/88186997/0143593e-4946-4630-967d-c490ea5dc84e)

**Comment by razzeee** [comment]
Some problems:

1. I'm unsure the data exitst, at least not on the metainfo spec site of things
2. The list of packagers can be X long, which is a problem for UI and user understanding

**Comment by kekkoudesu** [comment]
Yeah, I'm not sure how this could be implemented in a great way with Github. You can't specify a main maintainer, so you'd need to take _all_ of them if you even found a way. Probably not workable at least the way it works now...

I think there is some genuine confusion about what "Unverified" means, but that would be easily solvable in another way.

For example, how about a tooltip like what is shown for Verified packages when you hover over the checkmark? It might say, "Unverified means this app is packaged by a third party unaffiliated with the original developer or publisher."

**Comment by razzeee** [comment]
Yes, adding the badge was a quick win. Adding the hover was planned, but nobody worked on it yet.
